duckduckgo/iOS
# sync fetch devices from the server

Task/Issue URL: https://app.asana.com/0/392891325557410/1204428395509442
Tech Design URL:
CC:

**Description**:

Fetches devices from the server on the management screen.

**Steps to test this PR**:
1. Turn on sync
2. Current device should be shown
3. Close and open UI
4. Shows device
5. Add another device
6. Close and open UI to see updated list



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
